src/config/backends/index.ts

```diff
@@ -3,9 +3,20 @@ import { ExtraUEnv } from './extra-uEnv';
 import { ConfigTxt } from './config-txt';
 import { ConfigFs } from './config-fs';
 
-export default [
+export const all = [
 	new Extlinux(),
 	new ExtraUEnv(),
 	new ConfigTxt(),
 	new ConfigFs(),
 ];
+
+export function matchesAnyBootConfig(envVar: string): boolean {
```

Comment:
Is there a better way to write this ? I don't like how I have a variable outside the loop and the loop updates the value to true if one of the conditions passes. I want a way to pass a single value and a list of compare function. It's like the inverse of filter where it takes a list of values and compare function

Reply:
`_.some` would work here

Reply:
It looks like `_.some` is used as "here is a list of values, check if one of them returns true with my function".